libs/langchain/langchain/llms/fireworks.py

```diff
@@ -23,19 +25,21 @@ def _stream_response_to_generation_chunk(
     )
 
 
-class Fireworks(LLM):
+class Fireworks(BaseLLM):
     """Fireworks models."""
 
     model: str = "accounts/fireworks/models/llama-v2-7b-chat"
     model_kwargs: dict = Field(
         default_factory=lambda: {
-            "temperature": 0.7,
+            "temperature": 0.1,
```

Comment:
Why are we changing default temperature and top_p here? It could cause unexpected changes for callers who are using these defaults.